spec/lib/tasks/forms_admin_spec.rb

```diff
@@ -15,7 +15,7 @@
       Rake::Task["forms_admin:make_unlive"].invoke(form.id)
 
       form.reload
-      expect(form.has_live_version).to be false
+      expect(form.made_live_forms.present?).to be false
```

Comment:
It looks like forms that we make unlive with the Rake task/`make_unlive` endpoint will stay in the `live` state now. Although forms that were made unlive change from `draft` to `live` state before #425 was merged should stay in the `draft` state at least.